guides/essences.md

```diff
@@ -13,6 +13,85 @@ Alchemy comes with a lot of predefined essences for the regular needs of a websi
 
 Essences are Rails models. It is pretty easy to [add your own essence class](/create_essences.html) as well.
 
+## Rendering essences within your element
+
+Similar to [rendering elements within layouts](/elements.html#render_elements_in_your_layout), essences are rendered on an element. However, essences don't have their own partial views by default.
+
+Page layouts use the `<%= render_elements %>` helper to load an elements partial, elements use the `<%= element_view_for() do ... end %>` helper to expose a block that makes it easy to access the essences (instances of `Alchemy::Content`) of the element.
+
+For example. With an `article` element like this:
+
+~~~ yaml
+# config/alchemy/elements.yml
+- name: article
+  contents:
+    - name: headline
+      type: EssenceText
+    - name: color
+      type: EssenceText
+      settings:
+        input_type: color
+    - name: copy
+      type: EssenceRichtext
+~~~
+
+The `_article.html.erb` template generated by `rails alchemy:elements --skip` would look like this:
+
+~~~erb
+<%- cache(article) do -%>
+  <%= element_view_for(article) do |el| -%>
+    <div class="headline">
+      <%= el.render :headline %>
+    </div>
+    <div class="color">
+      <%= el.render :color %>
+    </div>
+    <div class="copy">
+      <%= el.render :copy %>
+    </div>
+  <%- end -%>
+<%- end -%>
+
+~~~
+
+Note how `element_view_for` allows you to call `el.render` on essences within the block.
+
+::: tip
+`element_view_for` wraps the block in a `<div>` with a set ID and class by default, but this is [customizeable](https://github.com/AlchemyCMS/alchemy_cms/blob/main/app/helpers/alchemy/elements_block_helper.rb#L137) if you pass in arguments:
+
+`<%= element_view_for(article, tag: :span, id: 'custom_id', class: 'custom_class') do |el| -%>`
+:::
+
+Without using the `element_view_for` helper, you can still access essences:
+
+`article.contents.find_by(name: 'headline')&.ingredient`
```

Comment:
There is even a shortcut method

```suggestion
`article.content_by_name('headline')&.ingredient`
```

And please use a code block for formatting

Reply:
Fixed